rq/job.py

```diff
@@ -1378,6 +1420,19 @@ def execute_failure_callback(self, death_penalty_class: Type[BaseDeathPenalty],
             logger.exception(f'Job {self.id}: error while executing failure callback')
             raise
 
+    def execute_stopped_callback(self, death_penalty_class: Type[BaseDeathPenalty]):
+        """Executes stopped_callback with possible timeout"""
+        if not self.stopped_callback:
```

Comment:
Can we move this check out of `execute_stopped_callback()`? So it should look like this:
```python
if job.stopped_callback:
    job.execute_stopped_callback()
```

Reply:
Done